HDSAviz/data_processing.py

```diff
@@ -0,0 +1,197 @@
+""" Tools for reading and processing the sensitivity analysis data files."""
```

Comment:
I agree - let's make a point to talk about this, and about how we want to format functions, etc tomorrow in class.